Source/ARTAuth.h

```diff
@@ -43,13 +43,17 @@ ART_ASSUME_NONNULL_BEGIN
  */
 - (void)requestToken:(art_nullable ARTTokenParams *)tokenParams withOptions:(art_nullable ARTAuthOptions *)authOptions
             callback:(void (^)(ARTTokenDetails *__art_nullable, NSError *__art_nullable))callback;
+- (void)requestToken:(void (^)(ARTTokenDetails *__art_nullable, NSError *__art_nullable))callback;
```

Comment:
Objective-C doesn't allow optional values on arguments, so we need to have another method.